.github/workflows/lint.yaml

```diff
@@ -25,6 +25,10 @@ on:
         description: 'Base 64-encoded content of the agent-version.cache file'
         required: false
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}
+  cancel-in-progress: ${{ github.event_name == 'pull_request' && true || false }}
```

Comment:
why not just `true`? Could it be possible to have concurrency with `workflow_dispatch`?

Reply:
Yes it's possible to manually trigger back-to-back runs.